packages/api/src/schema/api-schema.yaml

```diff
@@ -2923,6 +2932,56 @@ paths:
               schema:
                 $ref: "#/components/schemas/error"
   "/room/{id}/user/{userId}":
+    get:
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: userId
+          in: path
+          required: true
+          schema:
+            type: string
+      summary: Get user details
+      responses:
+        "200":
+          description: Success
```

Comment:
Can you add the response schema here as well? (`content` field)

You don't even need to define it as a separate component above if you don't already have it (tho fine to create it if you prefer). You can inline it like new assets endpoint: https://github.com/livepeer/studio/pull/1774/files#diff-bc9d02c3e9f64abe5a66b2dff920f42d87e200509c0b087384b1644bfad52c2cL2744